docs/reference/model_configuration.md

```diff
@@ -94,29 +94,39 @@ Configuration options for [`INCREMENTAL_BY_TIME_RANGE` models](../concepts/model
 
 Python model kind `name` enum value: [ModelKindName.INCREMENTAL_BY_TIME_RANGE](https://sqlmesh.readthedocs.io/en/stable/_readthedocs/html/sqlmesh/core/model/kind.html#ModelKindName)
 
+#### Incremental by partition
+
+Configuration options for [`INCREMENTAL_BY_PARTITION` models](../concepts/models/model_kinds.md#incremental_by_partition) (in addition to [general model properties](#general-model-properties) and [incremental model properties](#incremental-models)).
+
+| Option           | Description                                                                    | Type              | Required |
+| ---------------- | ------------------------------------------------------------------------------ | ----------------- | -------- |
+| `partitioned_by` | The column(s) and/or column expressions used define a model's partitioning key | str \| array[str] | Y        |
```

Comment:
Doesn't this make it seem like this is a property of a kind when in fact this should be specified at the model level?